js/fixToc.js

```diff
@@ -0,0 +1,65 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const SCROLL_OFFSET = 80; // Adjust based on your header height
+  const TOC_SELECTOR_LEFT = ".md-nav__link"; // Left nav links
+  const TOC_SELECTOR_RIGHT = ".md-nav--right .md-nav__link"; // Right nav links (update selector as needed)
+
+  // Smooth scroll with offset
+  function scrollToHash(hash) {
```

Comment:
This is just a nitpick but we're using two different ways to define functions, I'm sure I've probably done this a million times but for whatever reason it stood out to me 😬 haha. Can we stick to one?